src/App.tsx

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import BlogPosts from "./components/BlogPosts";
-import Features from "./components/Features";
-import JoinUs from "./components/JoinUs";
-import Main from "./components/Main";
-import Navbar from "./components/Navbar";
+import React from 'react';
+import BlogPosts from './components/BlogPosts';
+import Features from './components/Features';
+import JoinUs from './components/JoinUs';
+import Main from './components/Main';
+import Navbar from './components/Navbar';
```

Comment:
I am not sure why you changed all " to ', but I guess a formatter that you installed did.
If you are going to them  " to ', we need to change them all across the file, not just here.
So, I suggest to revert them back.

Reply:
- [ ] Revert the changes